src/fhirtypes/StructureDefinition.ts

```diff
@@ -205,6 +217,59 @@ export class StructureDefinition {
     return matchingElements.length === 1 ? matchingElements[0] : undefined;
   }
 
+  /**
+   * This function sets an instance property of an SD if possible
+   * @param {string} path - The path to the ElementDefinition to fix
+   * @param {FixedValueType} value - The value to fix
+   * @param {ResolveFn} resolve - A function that can resolve a type to a StructureDefinition instance
+   */
+  setInstancePropertyByPath(
+    path: string,
+    value: FixedValueType,
+    resolve: ResolveFn = () => undefined
+  ): void {
```

Comment:
So... I'm realizing now that this approach will work well for many parts of the SD, but it will _not_ work for `snapshot` or `differential` (or any of their child paths) since we abstract that to `elements`.  For now, I'm going to say that's OK -- and in fact, we should embrace it; throw an error if the path starts with `snapshot` or `differential`.  To set instances on those, they should be using the FSH path to identify the element and then caret path after that.